ui/src/pages/version/components/forms/components/ResourcesPanel.js

```diff
@@ -8,26 +14,87 @@ import {
   EuiFormRow,
   EuiSpacer,
   EuiCallOut,
+  EuiSuperSelect,
 } from "@elastic/eui";
 import { FormLabelWithToolTip, useOnChangeHandler } from "@caraml-dev/ui-lib";
 import { Panel } from "./Panel";
 import { calculateCost } from "../../../../../utils/costEstimation";
+import EnvironmentsContext from "../../../../../providers/environments/context";
 
 const maxTicks = 20;
 
 export const ResourcesPanel = ({
+  environment,
```

Comment:
Late to the party but shall we control the display of GPU options with a separate prop like `isGPUEnabled` and set it to true when the `<ResourcesPanel>` is created from `ModelStep.js` but leave it unset for `TransformerStep.js`?

Right now, the `environment` value is controlling whether we show GPU options or not which is not very clear immediately.

Reply:
Thanks for catching this, Krithika! We will update it accordingly! cc: @tkpd-hafizhan